### DISCUSSION TITLE
How to display on Apple Watch

### DISCUSSION THREAD

**Discussion by geoffschultz** [sbender9/WilhelmSKSupport#39] [Q&A]
I have data being displayed on my iPhone, but I can't figure out how to get it to display on my Apple Watch. From what I can tell, there's no app to install on the watch.  I'm sure that this is simple, but I'm baffled.

-- Geoff

**Comment by sbender9** [comment]
What version of WatchOS are you running. I made a mistake and it only runs on v11 right now. 

I have a version in test that runs on WatchOS v10.

**Comment by oregonjoe** [comment]
Trying to figure out how the apple watch function works - it it just a mirror of the iphone screens or is it a completely separate connection and configuration. We have a engine dashboard built on iPhone with live nmea2000 data from seagauge4 via TCP but only get blank screens on the apple watch
